schemas/manifest.schema.json

```diff
@@ -0,0 +1,473 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://github.com/YunoHost/apps/blob/master/schemas/manifest.schema.json",
+    "title": "Yunohost app manifest.toml schema",
+    "version": "0",
+
+    "type": "object",
+    "$defs": {
+        "translated_string": {
+            "type": "object",
+            "required": ["en"],
+            "additionalProperties": false,
+            "patternProperties": {
+                "^[a-z]{2}$": {
+                    "type": "string"
+                }
+            }
+        },
+        "byte_size": {
+            "type": "string",
+            "pattern": "^[0-9]*(\\.[0-9]*)?[kKmMgG]$"
+        },
+        "boolean_or_not_relevant": {
+            "anyOf": [
+                {
+                    "type": "boolean"
+                },
+                {
+                    "type": "string",
+                    "const": "not_relevant"
+                }
+            ]
+        },
+        "string_or_array_of": {
+            "anyOf": [
+                {
+                    "type": "string"
+                },
+                {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                }
+            ]
+        }
+    },
+
+    "properties": {
+        "packaging_format": {
+            "description": "Packaging format / version",
+            "type": "integer"
+        },
+        "id": {
+            "description": "App ID",
+            "type": "string"
+        },
+        "name": {
+            "description": "App Name",
+            "type": "string"
+        },
+        "version": {
+            "description": "App and package version",
+            "type": "string",
+            "pattern": "^[0-9\\.]*~ynh[0-9]*"
```

Comment:
Ogod there's a few ones not passing this because they have letters and dashes and stuff ... such as gamja, forgejo 

I was thinking "let's use the one from the linter" but it's absolutely horrendous : https://github.com/YunoHost/package_linter/blob/master/package_linter.py#L379

So idk watdo x_x

Reply:
Let's go with a simpler regex: `^[0-9]{1-3}([.-][0-9a-z]*~ynh[0-9]*$`